### ISSUE TITLE
Unable to run tests on jvm target in project that includes this library

### ISSUE THREAD

**Issue by wojta** [ionspin/kotlin-multiplatform-libsodium#26]
When I use it in my project, I'm unable to run tests with JUnit on JVM target. JS works fine.

I'm getting this error:
`
Execution failed for task ':crypto:jvmTest'.
> Could not resolve all files for configuration ':crypto:jvmTestRuntimeClasspath'.
   > Could not resolve org.jetbrains.kotlin:kotlin-test-junit5:1.7.10.
     Required by:
         project :crypto > org.jetbrains.kotlin:kotlin-test:1.7.10
      > Module 'org.jetbrains.kotlin:kotlin-test-junit5' has been rejected:
           Cannot select module with conflict on capability 'org.jetbrains.kotlin:kotlin-test-framework-impl:1.7.10' also provided by [org.jetbrains.kotlin:kotlin-test-junit:1.6.20-RC2(junitRuntime)]
   > Could not resolve org.jetbrains.kotlin:kotlin-test-junit:1.6.20-RC2.
     Required by:
         project :crypto > com.ionspin.kotlin:multiplatform-crypto-libsodium-bindings:0.8.6 > com.ionspin.kotlin:multiplatform-crypto-libsodium-bindings-jvm:0.8.6
      > Module 'org.jetbrains.kotlin:kotlin-test-junit' has been rejected:
           Cannot select module with conflict on capability 'org.jetbrains.kotlin:kotlin-test-framework-impl:1.6.20-RC2' also provided by [org.jetbrains.kotlin:kotlin-test-junit5:1.7.10(junit5Runtime)]

This is how I include it:
```
sourceSets {
        val coroutinesVersion: String by System.getProperties()
        val kotlinVersion: String by System.getProperties()
        val commonMain by getting {
            dependencies {
                implementation(project(":buffer"))
                implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:$coroutinesVersion")
                implementation("org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion")
                implementation("com.ionspin.kotlin:multiplatform-crypto-libsodium-bindings:0.8.6")

            }
        }
        val commonTest by getting {
            dependencies {
                implementation(kotlin("test"))
            }
        }
        val jvmMain by getting {
            dependencies {
            }
        }
        val jvmTest by getting
        val jsMain by getting {
            dependencies {
                implementation(npm("js-base64", "^3.7.2", generateExternals = true))
                implementation(npm("@peculiar/webcrypto", "^1.4.0", generateExternals = true))
            }

        }

        val jsTest by getting
    }
```
    
I think that it's caused by `Deps.Common.test` included in `commonMain` of the library. Wdyt? Is it necessary for something?
https://github.com/ionspin/kotlin-multiplatform-libsodium/blob/main/multiplatform-crypto-api/build.gradle.kts#L121

**Comment by ionspin** [comment]
Hi @wojta, thanks for reporting this, I'll have a look tomorrow and try to replicate. So far I have been running the tests successfully in a multiplatform project with 1.7.0, but haven`t tried with 1.7.10. 

And I'll try to remember why I needed the tests to be included in `commonMain` maybe it was a workaround for some problem, but I can't remember of the top of my head.

**Comment by wojta** [comment]
Found workaround
```
 implementation("com.ionspin.kotlin:multiplatform-crypto-libsodium-bindings:0.8.6") {
                    exclude("org.jetbrains.kotlin", "kotlin-test-junit")
                }
```

**Comment by ionspin** [comment]
I didn't try to replicate the issue and just went ahead and removed the test dependency from common source set, and bumped up the versions at the same time. If the CI finishes successfully it should be available in 0.8.7-SNAPSHOT in a few hours. If you still have problems, please reopen this issue! Once again thanks for reporting and providing a workaround!

**Comment by wojta** [comment]
Yes, it works without workaround. Thanks.
